client/src/nonblocking/rpc_client.rs

```diff
@@ -580,6 +581,33 @@ impl RpcClient {
         Ok(request)
     }
 
+    #[allow(deprecated)]
+    async fn maybe_map_filters(
+        &self,
+        mut filters: Vec<RpcFilterType>,
+    ) -> Result<Vec<RpcFilterType>, RpcError> {
+        let node_version = self.get_node_version().await?;
+        if node_version < semver::Version::new(1, 11, 2) {
```

Comment:
do we need to consider 1.10.30 here as well?  or maybe just change the docs to 1.11.2 and don't backport :sweat_smile: